### PR TITLE
Add GitHub Marketplace link to footer

### PR DESCRIPTION
**Pull request by abinoda**
## tldr

This is a new pull request based on feedback in https://github.com/integrations/slack/pull/768. The changes in this PR are:

* This PR adds a subtle "Browse other Slack integrations" link to the *footer* beside the support links.
* The link in this pull request uses a search query for Slack so as to not include any non-Slack integrations.

## Description

Every user has unique needs and preferences for Slack integration, and no single tool can solve them all. The GitHub Marketplace Actions<sup>1</sup> and Apps for Slack that enable a lot of custom functionality.

It's evident that users lack awareness of Marketplace, Actions, etc since this repo receives a lot of specific requests that could be solved with Actions<sup>2</sup> and Apps like Pull Reminders https://github.com/integrations/slack/issues/789.

Adding a link to Marketplace will help create awareness of more solutions, benefiting both end-users and GitHub by:

1. Steering users toward solutions so they don't get stuck or frustrated
2. Reducing support tickets for this project, many of which pertain to specific needs that can be addressed with GitHub Actions or Apps
3. Encouraging users to discover and get their hands dirty with Actions and Apps

<sup>[1]</sup> @nholden's [recent blog post](https://dev.to/nholden/how-i-replaced-a-rails-app-with-a-few-dozen-lines-of-ruby-3l1m) is a great example how an entire custom Slack bot can be re-implemented as a simple GitHub Action.
<sup>[2]</sup> In response to feature requests from Pull Reminders users, I have been recommending GitHub Actions and the results have been 💯 

## Screenshots

screen shot 2019-02-25 at 12 42 57 pm

screen shot 2019-02-25 at 12 40 32 pm

### CODE REVIEW

**Comment by scepticulous** [comment]
Closing it due to the missing PR branch. But we can add this any time if it still makes sense.
